src/services/EdFi.DataManagementService.Api.Tests/ApiSchemaBuilder.cs

```diff
@@ -3,8 +3,11 @@
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
 
+
+using System.Text.Json;
```

Comment:
Jim McKay would tell you to remove the extra line break above. Its not a big deal to me though 😆. `dotnet format` might be a good idea on this project, as a last commit though.